src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/stylesheet.css

```diff
@@ -15,7 +15,11 @@
     --code-font-family: 'DejaVu Sans Mono', monospace;
     /* Base font sizes for body and code elements */
     --body-font-size: 14px;
+    --block-font-size: 14px;
```

Comment:
This is a general comment about the entire file; this is just the first place I could attach a comment.

Wow. Impressive stuff. 
TIL: lots.  
I had to read up on some of the new constructs in use.  It's also much easier to read than the format of the file in times past, so all that Tech-Debt cleanup has been worth-while.